ui/apps/everest/src/components/cluster-form/resources/resources.tsx

```diff
@@ -1,33 +1,46 @@
-import { useEffect, useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useFormContext } from 'react-hook-form';
 import {
   Accordion,
   AccordionSummary,
   Box,
   Divider,
   FormGroup,
+  FormHelperText,
   InputAdornment,
+  Paper,
+  PaperProps,
   Stack,
+  SxProps,
   Typography,
   useTheme,
 } from '@mui/material';
 import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
-import { TextInput, ToggleButtonGroupInput, ToggleCard } from '@percona/ui-lib';
+import {
+  TextInput,
+  ToggleButtonGroupInput,
+  ToggleCard,
+  ToggleRegularButton,
+  ToggleButtonGroupInputRegular,
+} from '@percona/ui-lib';
 import { useKubernetesClusterResourcesInfo } from 'hooks/api/kubernetesClusters/useKubernetesClusterResourcesInfo';
 import { useActiveBreakpoint } from 'hooks/utils/useActiveBreakpoint';
 import {
   CUSTOM_NR_UNITS_INPUT_VALUE,
+  DEFAULT_CONFIG_SERVERS,
   DEFAULT_SIZES,
+  getDefaultNumberOfconfigServersByNumberOfNodes,
   humanizedResourceSizeMap,
+  MIN_NUMBER_OF_SHARDS,
   NODES_DB_TYPE_MAP,
   ResourceSize,
-  SHARDING_DEFAULTS,
 } from './constants';
 import { DbWizardFormFields } from 'consts';
 import { DbType } from '@percona/types';
 import { getProxyUnitNamesFromDbType } from './utils';
 import { ResourcesTogglesProps, ResourceInputProps } from './resources.types';
 import { Messages } from './messages';
+import { Theme } from '@emotion/react';
```

Comment:
I think this is the wrong theme. Should be the one from MUI

Reply:
fixed in https://github.com/percona/everest/pull/746/commits/e10cef70b608f1c8b6169813e10e2b54dac6b435